### PR TITLE
Add org.gtkhash.gtkhash

### PR DESCRIPTION
**Pull request by tristanheaven**
Add org.gtkhash.gtkhash

Known issue: GtkFileChooserNative doesn't allow access to files in the same directory as the one selected by the user.

### CODE REVIEW

File: org.gtkhash.gtkhash.yaml [hunk 1]
```diff
@@ -0,0 +1,35 @@
+app-id: org.gtkhash.gtkhash
```
Comment (bilelmoussaoui) [inline org.gtkhash.gtkhash.yaml:1]: The `gtkhash.org` doesn't seem to exists. You should use `com.github.tristanheaven.gtkhash` or another domain name you own.

File: org.gtkhash.gtkhash.yaml [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+command: gtkhash
+
+rename-desktop-file: gtkhash.desktop
+rename-appdata-file: gtkhash.appdata.xml
+rename-icon: gtkhash
+copy-icon: true # GtkAboutDialog uses logo-icon-name:PACKAGE
```
Comment (bilelmoussaoui) [inline org.gtkhash.gtkhash.yaml:10]: You should rename those upstream instead

File: org.gtkhash.gtkhash.yaml (hunk 1 above)
Comment (tristanheaven) [reply org.gtkhash.gtkhash.yaml:1]: It's registered by me, just doesn't have any A records.

File: org.gtkhash.gtkhash.yaml (hunk 2 above)
Comment (tristanheaven) [reply org.gtkhash.gtkhash.yaml:10]: Will do for next release.

**Comment by bilelmoussaoui** [comment]
bot, build org.gtkhash.gtkhash

**Comment by hfiguiere** [comment]
Appdata doesn't validate. Missing "oars" and "release"

**Comment by tristanheaven** [comment]
Thanks. I think the appdata and file renaming is all fixed in upstream git now.

Still need to do something about this xdg portal issue though as it breaks an important feature.

The application expects to be able to read files relative to the one selected through GtkFileChooserNative. Should GtkFileChooser be used instead for this?

**Comment by bilelmoussaoui** [comment]
> Thanks. I think the appdata and file renaming is all fixed in upstream git now.
> 
> Still need to do something about this xdg portal issue though as it breaks an important feature.
> 
> The application expects to be able to read files relative to the one selected through GtkFileChooserNative. Should GtkFileChooser be used instead for this?
The portal doesn't allow to achieve something similar currently. I would suggest using GtkFileChooser for now.

bot, build org.gtkhash.gtkhash

**Comment by tristanheaven** [comment]
Should work.

File: org.gtkhash.gtkhash.yaml [hunk 3]
```diff
@@ -4,32 +4,28 @@ runtime-version: '19.08'
...
 
-rename-desktop-file: gtkhash.desktop
-rename-appdata-file: gtkhash.appdata.xml
-rename-icon: gtkhash
-copy-icon: true # GtkAboutDialog uses logo-icon-name:PACKAGE
-
 finish-args:
   # X11 + XShm access
   - --share=ipc
   - --socket=x11
```
Comment (bilelmoussaoui) [inline org.gtkhash.gtkhash.yaml:10]:
```suggestion
```

**Comment by bilelmoussaoui** [comment]
bot, build org.gtkhash.gtkhash

**Pull request closed without merging**